packages/compass-components/package.json

```diff
@@ -29,74 +29,73 @@
-    "@leafygreen-ui/menu": "^29.0.5",
-    "@leafygreen-ui/modal": "^17.0.2",
-    "@leafygreen-ui/palette": "^4.1.3",
-    "@leafygreen-ui/pipeline": "^7.0.2",
-    "@leafygreen-ui/polymorphic": "^2.0.5",
-    "@leafygreen-ui/popover": "^13.0.11",
-    "@leafygreen-ui/portal": "^6.0.2",
-    "@leafygreen-ui/radio-box-group": "^14.0.2",
-    "@leafygreen-ui/radio-group": "^12.0.2",
-    "@leafygreen-ui/search-input": "^5.0.2",
-    "@leafygreen-ui/segmented-control": "^10.0.2",
-    "@leafygreen-ui/select": "^14.0.2",
-    "@leafygreen-ui/skeleton-loader": "^2.0.11",
-    "@leafygreen-ui/split-button": "^4.1.5",
-    "@leafygreen-ui/table": "^13.0.1",
-    "@leafygreen-ui/tabs": "^17.0.4",
-    "@leafygreen-ui/text-area": "^10.0.2",
-    "@leafygreen-ui/text-input": "^14.0.2",
-    "@leafygreen-ui/toast": "^7.0.2",
-    "@leafygreen-ui/toggle": "^11.0.2",
-    "@leafygreen-ui/tokens": "^3.2.4",
-    "@leafygreen-ui/tooltip": "^13.0.13",
-    "@leafygreen-ui/typography": "^20.0.2",
+    "@leafygreen-ui/avatar": "^3.1.4",
+    "@leafygreen-ui/badge": "^10.2.2",
+    "@leafygreen-ui/banner": "^10.2.2",
+    "@leafygreen-ui/button": "^25.1.2",
+    "@leafygreen-ui/card": "^13.2.0",
+    "@leafygreen-ui/checkbox": "^18.1.2",
+    "@leafygreen-ui/chip": "^4.0.8",
+    "@leafygreen-ui/code": "^20.2.1",
+    "@leafygreen-ui/combobox": "^12.1.0",
+    "@leafygreen-ui/confirmation-modal": "^8.0.0",
+    "@leafygreen-ui/copyable": "^12.0.0",
+    "@leafygreen-ui/drawer": "^5.1.2",
+    "@leafygreen-ui/emotion": "^5.1.0",
+    "@leafygreen-ui/guide-cue": "^8.1.0",
+    "@leafygreen-ui/hooks": "^9.2.2",
+    "@leafygreen-ui/icon": "^14.6.1",
+    "@leafygreen-ui/icon-button": "^17.1.2",
+    "@leafygreen-ui/info-sprinkle": "^5.0.8",
+    "@leafygreen-ui/input-option": "^4.1.2",
+    "@leafygreen-ui/leafygreen-provider": "^5.0.4",
+    "@leafygreen-ui/logo": "^11.1.0",
+    "@leafygreen-ui/marketing-modal": "^6.0.0",
+    "@leafygreen-ui/menu": "^33.0.0",
+    "@leafygreen-ui/modal": "^18.0.0",
+    "@leafygreen-ui/palette": "^5.0.2",
+    "@leafygreen-ui/pipeline": "^8.0.7",
+    "@leafygreen-ui/polymorphic": "^3.1.0",
+    "@leafygreen-ui/popover": "^14.3.0",
+    "@leafygreen-ui/portal": "^7.1.0",
+    "@leafygreen-ui/radio-box-group": "^15.0.9",
+    "@leafygreen-ui/radio-group": "^13.0.9",
+    "@leafygreen-ui/search-input": "^6.1.0",
+    "@leafygreen-ui/segmented-control": "^11.0.10",
+    "@leafygreen-ui/select": "^16.3.0",
+    "@leafygreen-ui/skeleton-loader": "^3.0.8",
+    "@leafygreen-ui/split-button": "^6.1.6",
+    "@leafygreen-ui/table": "^15.2.0",
+    "@leafygreen-ui/tabs": "^17.0.6",
+    "@leafygreen-ui/text-area": "^12.1.2",
+    "@leafygreen-ui/text-input": "^16.1.3",
+    "@leafygreen-ui/toast": "^8.1.0",
+    "@leafygreen-ui/toggle": "^12.1.2",
+    "@leafygreen-ui/tokens": "^4.0.0",
+    "@leafygreen-ui/tooltip": "^14.2.2",
+    "@leafygreen-ui/typography": "^22.2.0",
     "@lg-chat/avatar": "^7.0.2",
```

Comment:
Are those not "non-modal" components? Looks like latest is 8, not 7. Same question for some other lg-chat stuff, curious why we're not updating those too

Reply:
You're right - good call-out.

Short answer is I believe these new versions were released since I branched off and I didn't want to complicate things by upgrading these in the process as well. I imagine I'll need to circle back on these once my modal upgrading PR lands 🙄

Reply:
BTW - these also got renamed from the `@lg-chat` to `@leafygreen-ui`.

Reply:
I renamed the PR title to reflect this.